frontends/web/src/utils/custom.d.ts

```diff
@@ -30,7 +30,7 @@ declare module '*.svg';
 declare namespace JSX { // tslint:disable-line:no-namespace
     interface HTMLAttributes {
         align?: 'left' | 'right' | 'center';
-        allow?: 'payment';
+        allow?: 'camera; payment';
```

Comment:
I think this should just be an arbitrary optional string at this point. Otherwise, it's basically a copy&paste typo checker  :)

Reply:
true! done

Reply:
:+1: